Week3/homework/step2-4.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
 function Dog() {
-  // add your code here
+  this.name = 'Puppy';
+  this.color = 'brown';
+  this.numLegs = 4;
 }
 
 const hound = new Dog();
+hound.name = 'Miami';
+hound.color = 'pink';
```

Comment:
I understand, so intstead of writing too long code, I can make it shorter as:
```suggestion
function betterDog(name, color, numLegs) {
  this.name = name;
  this.color = color;
  this.numLegs = numLegs;
}
```
So that, I can call them in only one small function.
```suggestion
const betterHound = new betterDog('betterMiami', 'betterPink', 4);
```
This is what you said, right?

Reply:
Because I don't know how to make suggestion on GitHub, I marked only line of `hound.color = 'pink';`. But I wanted to make suggestions for line 3-7 and line 9-11.